extensions/amp-accordion/1.0/accordion.js

```diff
@@ -61,6 +61,7 @@ function AccordionWithRef(
     as: Comp = 'section',
     expandSingleSection = false,
     animate = false,
+    experimentDisplayLocking = false,
```

Comment:
Top level Preact prop, wanted the name to be closely matched with amp experiment name which is currently set to `amp-accordion-display-locking` from `0.1`